frontend/src/workspace/buildingtools/FunNode.js

```diff
@@ -78,15 +79,23 @@ function FunNode(props) {
   const y = props.y;
```

Comment:
Since you are only exporting FunNode, which is particularly large, I think you can just do 
`export default function FunNode(props){....}` on line 75.